lab_1/main.py

```diff
@@ -25,7 +37,32 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    words = open('C:\\Users\\Acer\\2020-2-level-labs\\lab_1\\stop_words.txt', "r", encoding='utf-8')
```

Comment:
You have an argument `stop_words` - you should work with it, i.e. filter the out from the `tokens` - another argument.